easybuild/easyconfigs/g/goolf/goolf-1.5.16.eb

```diff
@@ -0,0 +1,37 @@
+easyblock = "Toolchain"
+
+name = 'goolf'
+version = '1.5.16'
```

Comment:
plus, leaving it as-is keeps it aligned with `gompi/1.5.16` which is a good thing, since they are tested together :+1: